### PR TITLE
Update Japanese translations

### PR DESCRIPTION
**Pull request by miurahr**
Update Japanese translations from OmegaT-L10n/ja project.
Now we are running validation process, so add WIP.

### CODE REVIEW

File: _i18n/ja.yml [hunk 1]
```diff
@@ -1,77 +1,63 @@
...
+     docx: Howto - Word 2007 ファイルを OmegaT に翻訳する
+     export_uncleaned_rtf: Howto - OmegaT から「Trados uncleaned RTF」フォーマットへのエクスポート
+     google_translate: Howto - Google Translate を OmegaT で利用する
+     iceni_infix: Howto - Iceni infixとOmegaTによるPDFファイルの翻訳
+     linux: Linux上のOmegaT 技術情報
+     localizing_omegat: Howto - OmegaT をご使用の言語に翻訳する
+     ms_translator: Howto - Microsoft Translator を OmegaT で利用する
+     new_filter: Howto-新しいファイルフィルターを作成する
+     preview: Howto - OmegaT へプレビュー機能を追加する
+     priorview: Howto - 「Priorview」プレビュー・ユーティリティーを利用する
```
Comment (miurahr) [inline _i18n/ja.yml:47]: It should be obsoleted

File: _i18n/ja/about.html [hunk 2]
```diff
@@ -1,101 +1,49 @@
...
+  <div class="text-md-center">
+    <h3>Webサイトのコンテンツ作成者</h3>
+    <ul class="list-unstyled">
+      {% for author in site.data.credits.website %}
+        <li>{{ author }}</li>
+      {% endfor %}
+    </ul>
+
+    {% if site.lang != "en" %}
+    <h3>英語版の翻訳者</h3>
```
Comment (miurahr) [inline _i18n/ja/about.html:38]: Here is now under discussion.

File: _i18n/ja.yml (hunk 1 above)
Comment (miurahr) [reply _i18n/ja.yml:47]: PR pushed on en.yml

**Comment by miurahr** [comment]
Now ready to merge.

**Review by amake — APPROVED** [review]

**Pull request merged**